src/Authentication/GraphPhpLeagueAccessTokenProvider.php

```diff
@@ -24,14 +27,37 @@
  */
 class GraphPhpLeagueAccessTokenProvider extends PhpLeagueAccessTokenProvider
 {
+    public const NATIONAL_CLOUD_TO_AZURE_AD_ENDPOINT = [
+        NationalCloud::GLOBAL => 'https://login.microsoftonline.com',
+        NationalCloud::US_GOV => 'https://login.microsoftonline.us',
+        NationalCloud::CHINA => 'https://login.chinacloudapi.cn'
+    ];
+
     /**
      * @param TokenRequestContext $tokenRequestContext
      * @param array<string> $scopes if left empty, it's set to ["https://[graph national cloud host]/.default"] scope
+     * @param string $nationalCloud Defaults to https://graph.microsoft.com. See
+     * https://learn.microsoft.com/en-us/graph/deployments
      */
-    public function __construct(TokenRequestContext $tokenRequestContext, array $scopes = [])
+    public function __construct(
+        TokenRequestContext $tokenRequestContext,
+        array $scopes = [],
+        string $nationalCloud = NationalCloud::GLOBAL
+    )
     {
         $allowedHosts = ["graph.microsoft.com", "graph.microsoft.us", "dod-graph.microsoft.us", "graph.microsoft.de",
```

Comment:
We should also `graph.microsoft-ppe.com` in the list

Reply:
Added.